### DISCUSSION TITLE
Support for footnotes

### DISCUSSION THREAD

**Discussion by Limezy** [outline/outline#2582] [Ideas]
Hi all, 

That would be very nice if Outline could support footnotes.

```
For example, the forum software Flarum uses the syntax^1
---
^1 : write your footnote here
```

When rendering the "1" will appear as superscript and will be clickable to go down to the footnote directly

**Comment by Limezy** [comment]
Hi, for those who would be interested, I've found a small "hack" to include footnotes into my Outline documents.
I just use the superscript numbers found in my special character palet : 
<img width="686" alt="image" src="https://user-images.githubusercontent.com/24638389/156546704-a3233a95-14aa-4bee-94e7-272ac3e9a057.png">

And then, I create a link on the superscript number, that forwards to a new document including my footnote.
The latest versions of Outline even provide a preview mode for links, which make my footnotes visible when flying over the superscript.

<img width="617" alt="image" src="https://user-images.githubusercontent.com/24638389/156547542-25e86105-8eec-432b-aad2-81eef099d3db.png">

I also make sure that the note document is nested under the master document 😄

**Comment by NoNameProvided** [comment]
I also would like to see this feature implemented. However, it may be better to align with the existing implementation in Github Flavored Markdown, [which uses a different syntax](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/):

```
Here is a simple footnote[^1]. With some additional text after it.

[^1]: My reference.
```

**Reply by Limezy** [reply]
Whatever syntax would be a great add to Outline !

**Reply by finsrud** [reply]
AGree!

**Comment by Konafets** [comment]
I want to push this idea.

**Comment by N07070** [comment]
+1 This would be very nice

**Comment by chillmatin** [comment]
Any updates? I need this feature.
